src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -1926,7 +1953,7 @@ HRESULT CEndlessUsbToolDlg::OnTryEndlessSelected(IHTMLElement* pElement)
 {
     FUNCTION_ENTER;
 
-	Analytics::instance()->eventTracking(_T(ELEMENT_FIRST_PAGE), _T("USBType"), _T("Live"));
+	eventTracking(_T("Selected"));
```

Comment:
Maybe it would be cleaner to define `::SetSelectedInstallMethod(InstallMethod_t method) { m_selectedInstallMethod = InstallMethod_t::TryEndless; eventTracking(_T("Selected")); }` and use this consistently?